src/main/java/com/exadel/discount/util/AccessTokenUtil.java

```diff
@@ -0,0 +1,63 @@
+package com.exadel.discount.util;
+
+import com.exadel.discount.model.token.TokenPayload;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
+public class AccessTokenUtil {
```

Comment:
No need to create 2 separate services for access and refresh

Reply:
This is done.